### ISSUE TITLE
Searching Algortihms

### ISSUE THREAD

**Issue by 4N1Z** [4N1Z/Sorting-Searching-Algorithms#3]
Points to be noted
- Need correct code for the any searching algorithms you upload.
- Push the code to the correct directory.
- Should also name the file as per the language of your code before you submit an PR.
- The codes will be checked thoroughly.

**Comment by Tejas20-03** [comment]
It will be my pleasure to contribute on above issue.

**Comment by AnirudhDaya** [comment]
Yes, go ahead. We will only merge a clean and good. So make sure you do so.

**Comment by HimangsKalita** [comment]
please check Added jump-search in C++ #8

**Comment by Atharva-D** [comment]
I have added Binary Search in C++ and Python please review and merge the changes.

**Comment by manishtambe** [comment]
I have added Quick Sort in C++ please merge the changes !

**Comment by sunilseervi6** [comment]
Can I do binary search in Java?

**Comment by Ankitrai2204** [comment]
Hello I want to contribute on this project. Can you please assign it to me and tell about our requirements.

**Comment by 4N1Z** [comment]
> Hello I want to contribute on this project. Can you please assign it to me and tell about our requirements.

Yes, you are free to contribute. Read the issues which are already given and you can contribute as per your knowledge  or create a new issue if you find any mistakes and we will see to it.

**Comment by 4N1Z** [comment]
> Can I do binary search in Java?

Binary Search in java already added, Try something else .

**Comment by sunilseervi6** [comment]
What algorithm can I use then
Can u please assign me what should I use?

**Comment by Souvik2024** [comment]
Hi @4N1Z, I want to contribute to this, Can you please assign me this?

**Comment by Rufus-paul** [comment]
can you please assign this issue to me

**Comment by shardendu930** [comment]
Hii @4N1Z I have added interpolation search in cpp . Please review my pr and merge it .

**Comment by Danish-Belal** [comment]
@4N1Z  Can I work on it  .

**Comment by rxhith** [comment]
Can  I do DFS and bfs using user input?

**Comment by Souvik2024** [comment]
Hi, @4N1Z  I have added BFS in python. Please review my pr and merge it

**Comment by Souvik2024** [comment]
Hey @4N1Z  please check I have added the BFS code in Python #186

**Comment by Souvik2024** [comment]
Hi @4N1Z, please check and review I have added the BFS code in Python #186

**Comment by paras-2407** [comment]
Hello, can you please assign me this project I would like to add codes in language except for the one already included in the repo.
